staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

```diff
@@ -258,6 +261,38 @@ func TestProxyHandler(t *testing.T) {
 			},
 			expectedStatusCode: http.StatusServiceUnavailable,
 		},
+		"change aggregator http request host": {
```

Comment:
what is the difference between this test case and "proxy with user, insecure"?

Reply:
Same as 'proxy with user, insecure',i think i can remove the test case.just use 'targetServer.Listener.Addr().String() equal target.host' to cover my change,is right?

Reply:
sounds good. 

could you try the following 1) remove your fix in staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go 2) run the test. The test should fail.

Reply:
According to your suggestion, I tried it and it runs as expected.